tools/crm_resource_runtime.c

```diff
@@ -1241,7 +1242,7 @@ cli_resource_restart(resource_t * rsc, const char *host, int timeout_ms, cib_t *
             goto failure;
         }
 
-    } while(g_list_length(list_delta) > 0);
```

Comment:
Right, it's actually two separate while loops, obfuscated by whitespace usage :)

Since they have the same condition, the second one is never true, so it has no noticeable effect.